src/OpenCL/Silk.NET.OpenCL/Enums/LoaderLayers.gen.cs

```diff
@@ -7,10 +7,12 @@
 
 #pragma warning disable 1591
 
-namespace Silk.NET.OpenCL.Extensions.APPLE
+namespace Silk.NET.OpenCL
 {
-    [NativeName("Name", "GLenum")]
-    public enum APPLE : int
```

Comment:
This is a breaking change. While we're happy to break if the upstream specification dictates it, we generally always have ungrouped enums available thus if it's possible I'd prefer to either:
- have the grouped enum in addition to the ungrouped vendor-wide enum
- stop this change entirely

If 1 is not too much trouble ;)

Reply:
Yeah, found the issue, those empty enums should be back in business now